scripts/go.mod

```diff
@@ -3,7 +3,7 @@ module _
 go 1.17
 
 require (
-	github.com/google/go-jsonnet v0.17.1-0.20210520122306-7373f5b60678
+	github.com/google/go-jsonnet v0.20.0
```

Comment:
Update needed to get `std.get()` working.

Reply:
I'm also updating this here: https://github.com/kubernetes-monitoring/kubernetes-mixin/pull/891